lib/output.js

```diff
@@ -186,6 +186,7 @@ function OutputStream(options) {
         webkit           : false,
         width            : 80,
         wrap_iife        : false,
+        wrap_func_args   : false,
```

Comment:
Please go ahead and make this true by default :) some things in Terser don't provide the least possible byte size and that's OK.